.circleci/config.yml

```
@@ -1,6 +1,23 @@
 version: 2
 
 jobs:
+  lint:
+    docker:
+      - image: replicated/dockerfilelint
+    working_directory: /data
+    environment:
+      DOCKERFILE_LIST: /tmp/dockerfiles.txt
+    steps:
+      - checkout
+      - run:
+          name: Lint dockerfiles
+          command: |
+            find * -name Dockerfile > ${DOCKERFILE_LIST}
+            while read DOCKERFILE; do
+              echo linting ${DOCKERFILE};
+              # @todo remove the || once lint issues all resolved.
+              /dockerfilelint/bin/dockerfilelint ${DOCKERFILE} || echo "${DOCKERFILE} failed linting - continuing"
```

I would just let it fail. That way we known when its fixed.